mmv1/third_party/terraform/tpgresource/common_diff_suppress.go.erb

```diff
@@ -115,6 +115,23 @@ func EmptyOrUnsetBlockDiffSuppress(k, old, new string, d *schema.ResourceData) b
 	return EmptyOrUnsetBlockDiffSuppressLogic(k, old, new, o, n)
 }
 
+func ListStringsSuppress(values []string) schema.SchemaDiffSuppressFunc {
+       return func(k, old, new string, d *schema.ResourceData) bool {
+               return (old == "" && implContains(values, new)) || (new == "" && implContains(values, old))
+       }
+}
```

Comment:
only nit: we use tabs instead of spaces here.

Reply:
updated